JSTutorial/hw_8/task_4/ITSpecialist.js

```diff
@@ -0,0 +1,30 @@
+class ITSpecialist {
+    #name
+    #grade
+    #experience_in_years
+    #age
+    #country
+    #_salary
```

Comment:
```suggestion
    #salary
```
You are already using private field, no need to add underscore